src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodsList = ({ goods }) => (
+  <>
+    <ul>
+      {goods.map(good => (
+        <li key={good}>
+          {good}
+        </li>
+      ))}
+    </ul>
+  </>
```

Comment:
```suggestion
    <ul>
      {goods.map(good => (
        <li key={good}>
          {good}
        </li>
      ))}
    </ul>
  ```

No need in the fragment here, Everything is already wrapped in 'ul' tag and it is the only tag here

Reply:
done